targets/psa_switch/pswitch_CLI.py

```diff
@@ -88,7 +88,43 @@ def main():
         args.thrift_ip, args.thrift_port, services
     )
 
-    runtime_CLI.load_json_config(standard_client, args.json)
+    json_ = runtime_CLI.load_json_config(standard_client, args.json)
+    def get_json_key(key):
+        return json_.get(key, [])
+
+    for j_meter in get_json_key("extern_instances"):
+        if j_meter["type"] != "Meter":
+            continue
+        meter_array = runtime_CLI.MeterArray(j_meter["name"], j_meter["id"])
+        attribute_values = j_meter.get("attribute_values", [])
+        for attr in attribute_values:
+            name = attr.get("name", [])
+            val_type = attr.get("type", [])
+            value = attr.get("value", [])
+            if name == "is_direct":
+                meter_array.is_direct = value == True
+            # TODO set meter_array.binding for direct_meter
+            # direct_meter not supported on PSA yet
```

Comment:
I think you mean "not supported by psa_switch yet", PSA implies the architecture :)